course_content/notebooks/dask_intro.ipynb

```diff
@@ -831,6 +831,94 @@
    "cell_type": "raw",
    "metadata": {},
    "source": []
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
So my first point is that all this content needs to go **above** the loading exercise; not below. You might then need to rebase to pull in the changes that @LukeC92 made, as you might get merge conflicts otherwise.